ompi/mpi/java/java/Comm.java

```diff
@@ -162,12 +162,12 @@ public Comm iDup() throws MPIException
 	 */
 	public Comm dupWithInfo(Info info) throws MPIException
 	{
-	    MPI.check();
-	    return new Comm(dupWithInfo(handle, info.handle));
+		MPI.check();
+		return new Comm(dupWithInfo(handle, info.handle));
```

Comment:
minor thing, and don't redo for this PR, but normally try to avoid whitespace changes in PRs that are not just whitespace changes.